src/bptl/work_units/kownsl/tasks.py

```diff
@@ -123,3 +146,112 @@ def get_approval_status(task: BaseTask) -> dict:
             "num_rejected": num_rejected,
         },
     }
+
+
+@register
+def get_review_response_status(task: BaseTask) -> dict:
+    """
+    Get the reviewers who have not yet responded to a review request so that
+    a reminder email can be sent to them if they exist.
+
+    In the task binding, the service with alias ``kownsl`` must be connected, so that
+    this task knows which endpoints to contact.
+
+    **Required process variables**
+
+    * ``reviewRequestId``: the identifier of the Kowns review request, used to update
```

Comment:
```suggestion
    * ``reviewRequestId``: the identifier of the Kownsl review request, used to update
```

Reply:
this documentation in general seems to be a bit off - there's no `update` happening, right?

Reply:
Youre right. Dumb copy paste error.